_shared/project/requirements/functests.in

```diff
@@ -9,6 +9,9 @@ h-matchers
 {% if cookiecutter.get("_directory") == "pyramid-app" %}
 webtest
 {% endif %}
+{% if cookiecutter.get("__parallel_unit_tests") and cookiecutter.get("postgres") == "yes" %}
+filelock
```

Comment:
The `filelock` dependency that's used for locking when creating the test DB when running the unit tests in parallel. We don't support running the functests in parallel yet but we might one day and in at least some projects the functtests and unit tests share the same `db_engine` fixture so the functests need `filelock` anyway